src/test/java/com/klarna/hiverunner/TimeoutAndRetryTest.java

```diff
@@ -153,4 +153,4 @@ public void endOnSecondRun() {
 
 
 
-}
+}
```

Comment:
Add a [closing line break](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to EOF please.